src/mscorlib/shared/System/Math.cs

```diff
@@ -63,6 +99,12 @@ public static decimal Abs(decimal value)
             return decimal.Abs(value);
         }
 
+        [StackTraceHidden]
```

Comment:
What does this do, prevent inlining?

Reply:
Stops it showing up in exception stack trace https://github.com/dotnet/coreclr/pull/14652 so it looks like the throw was in-place rather than off a throw helper function